Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonColorDialog.cs

```diff
@@ -34,16 +34,6 @@ public string Title
                 ClickCallback = ClickCallback
             };
 
-        /// <summary>
```

Comment:
Why is this removed

Reply:
It had a duplicate entry, so it wouldn't build